release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+
+The configurable widgets that displayed circle charts based on policy categories have been removed. To view similar data, you can use the *Policy violations by category* widget and select the number of violations in each category to display more information. 
+
+
+[id="network-policies-list"]
+=== List of network policies in Deployment tab for violations 
+
+When viewing a violation, you can select the *Deployment* tab and scroll to the *Network Policy* section to view a list of network policies in the namespace for the deployed cluster. For example, if you have configured the system to alert you of a violation of policy *Deployments should have at least one ingress network policy*, you can choose the *Deployment* tab to view a list of the existing network policies. You can use this list to verify which network policies exist in the namespace and determine whether or not an ingress network policy is missing. 
+
+[id="privilege-escalation"]
+=== Default policy for privilege escalation
+
+This new default policy detects if a deployment is running with a container that has `allowPrivilegeEscalation` set to `true`. This policy is enabled by default. The privilege escalation setting is enabled in Kubernetes pods by default. A container process that can run with more privileges than its parent process can allow the container to run with unintended privileges, creating a security risk. This policy provides an alert so that you can verify whether privileged escalation is required. For example, it may be necessary if the required privileges cannot be provided with a subset of other controls.
+
+[id="externally-exposed-service"]
+=== Default policy for externally exposed service
+
+This new policy detects if a deployment has any service that is externally exposed through any methods. The policy is disabled by default. Deployments with services exposed outside the cluster are at a higher risk of attempted intrusions because they are reachable outside of the cluster. This policy provides an alert so that you can verify that service exposure outside of the cluster is required. If the service is only needed for intra-cluster communication, use service type `ClusterIP`.
+
+[id="multiple-roles-user-group"]
+=== Ability to assign multiple RHACS roles to users and groups
+
+Role-based Access Control (RBAC) in {product-title-short} has been enhanced to allow you to assign multiple roles to a single user or group. Previously, a key/value pair could only be assigned to a single role. Now, that key/value pair can be assigned to multiple {product-title-short} roles. This change allows you to reuse existing roles and simplifies role management. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.71/html/operating/managing-user-access#manage-role-based-access-control[Managing RBAC in Red Hat Advanced Cluster Security for Kubernetes].
```

Comment:
Vale is telling me that key/value pair should be written as "key and value pair", but I couldn't find it in the IBMSG.

Reply:
It doesn't seem to have its own entry, but there is a reference to "key-value pair" here: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting  So I guess I could use that? I've definitely never seen it with "and" though.